src/register/register.ts

```diff
@@ -63,12 +84,17 @@ export class Register {
    * register ".
    */
   public static async get(vimState: VimState): Promise<IRegisterContent> {
-    const register = vimState.recordedState.registerName;
+    let register = vimState.recordedState.registerName;
 
     if (!Register.isValidRegister(register)) {
       throw new Error(`Invalid register ${register}`);
     }
 
+    // Register names are case-insensitive.
```

Comment:
This comment is a little bit misleading. Probably you want to say something like `retrieving register content is case-insensitive.`